ts/components/screens/OperationResultScreenContent.tsx

```diff
@@ -27,7 +27,7 @@ type OperationResultScreenContentProps = WithTestID<{
   subtitle?: string | Array<BodyProps>;
   action?: Pick<
     ButtonSolidProps,
-    "label" | "accessibilityLabel" | "onPress" | "testID"
+    "label" | "accessibilityLabel" | "onPress" | "testID" | "fullWidth"
```

Comment:
Why do you add `fullWidth` prop? According to design guidelines we don't need the full-width buttons
```suggestion
    "label" | "accessibilityLabel" | "onPress" | "testID"
```

Reply:
You're right, reverted